src/validators/identificationbirthplace.js

```diff
@@ -1,7 +1,15 @@
 /* eslint-disable import/prefer-default-export */
 import { validateModel } from 'models/validate'
-import identificationPlaceOfBirth from 'models/sections/identificationPlaceOfBirth'
+import birthplace from 'models/shared/locations/birthplace'
```

Comment:
This change should be made in the `models/sections/identificationPlaceOfBirth` file instead of this one.

Reply:
Ok, updated, but can you show me where/how this gets called?